### ISSUE TITLE
Preview Support?

### ISSUE THREAD

**Issue by MysticalOS** [italomandara/CXPatcher#205]
I thought they'd update preview more often, but as of now it actually can't play ANY ubisoft games do to ubisoft connect issue they only fixed in non preview. if preview isn't gonna get updated with hotfixes when stuff breaks, maybe this can perform that action? Assuming such fixes are possible (migrating from their patches back into preview and what not)

in addition to regular stuff of course, GPTK and dxvk udpating and the like.

**Comment by italomandara** [comment]
I'm not able to access CX Preview myself but if you do you can fork your own "Preview edition" of CXPatcher and test it

**Comment by MysticalOS** [comment]
interesting, so that's fix, then how does it work for steam games that require ubisoft connect, cause team requires a windows 10 bottle.

**Comment by MysticalOS** [comment]
it didn't, but I assumed it's cause I attempted it with a downgraded bottle from wine 9.6/preview, but maybe their fix just didn't work for that scenario. yeah I own a lot of ubisoft steam games, so that's a mess right there. Guess we'll have to see if they got a better fix forthcoming.. anyways I did post on their forums I'd love to see preview at least get hotfix updates in future as feedback. So Close this issue here.
